wp-content/themes/heideandcook/web/sass/_fancybox.scss

```diff
@@ -1,21 +1,14 @@
 .custom-close {
-    background-image: none;
-    top: rem-calc(-6px);
-    right: rem-calc(20px);
+    background-image: url("../images/close-icon.png");
+    background-size: rem-calc(30px);
```

Comment:
instead of background size set this as a block with a fixed width and height that is the same as the image.